src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -28,11 +30,13 @@ class AptitudePackageManager(PackageManager):
     # For more details, try `man apt-get` on any Debian/Ubuntu based box.
     def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, status_handler):
         super(AptitudePackageManager, self).__init__(env_layer, execution_config, composite_logger, telemetry_writer, status_handler)
+
+        security_list_guid = str(uuid.uuid4())
         # Repo refresh
         self.repo_refresh = 'sudo apt-get -q update'
 
         # Support to get updates and their dependencies
-        self.security_sources_list = '/tmp/az-update-security.list'
+        self.security_sources_list = execution_config.temp_folder + '/az-update-security-{0}.list'.format(security_list_guid)
```

Comment:
we should change name from "az-update-security-{0}" to "aumcenter-security-patches-{0}"
az-update-security is used in v1.

Reply:
The extension serves Azure scenarios and is not AUM specific - AUM is one consumer of the data that comes out the other end. It's tagged as such for that reason (az=Azure)

Reply:
Renamed this to "msft-patch-security-{0}.list" to keep it generic.